src/main/java/Duke.java

```diff
@@ -1,10 +1,88 @@
+import java.lang.reflect.Array;
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+
+    public static void promptUser(String message) {
+        System.out.println(message);
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        //initiate objects
+        Scanner sc = new Scanner(System.in);
+        ArrayList<Task> list = new ArrayList<>();
+        //read user input
+        while(true) {
+            String input = sc.nextLine();
+            String[] words = input.split(" ");
+            String firstWord = words[0];
+            if (input.equals("bye")){
```

Comment:
Maybe you can use equalsIgnoreCase so that "Bye" or "BYE" etc. can work as well.
But otherwise, I really like how neat this is!